cardano-cli/golden/files/golden/help.cli

```diff
@@ -1330,6 +1517,8 @@ Usage: cardano-cli shelley text-view decode-cbor --in-file FILE
 Usage: cardano-cli shelley governance 
```

Comment:
We still have `cardano-cli shelley ...` CLI command structure.  Is this something we want to retain going forward?